lldb/source/Target/StackID.cpp

```diff
@@ -61,6 +61,13 @@ bool lldb_private::operator<(const StackID &lhs, const StackID &rhs) {
   const lldb::addr_t lhs_cfa = lhs.GetCallFrameAddress();
   const lldb::addr_t rhs_cfa = rhs.GetCallFrameAddress();
 
+  // FIXME: rdar://76119439
```

Comment:
Can a async function call another async function that also gets detached?  In that case, there's really no natural ordering since they would both be in the heap and where is subject to the whims of malloc...